pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -206,16 +250,28 @@ func (rc *reconciler) reconcile() {
 			glog.V(5).Infof("Volume %q/Node %q is attached--touching.", volumeToAttach.VolumeName, volumeToAttach.NodeName)
 			rc.actualStateOfWorld.ResetDetachRequestTime(volumeToAttach.VolumeName, volumeToAttach.NodeName)
 		} else {
+			// Don't even try to start an operation if there is already one running
+			if rc.attacherDetacher.IsOperationPending(volumeToAttach.VolumeName, "") {
+				glog.V(10).Infof("Operation for volume %q is already running. Can't start attach for %q", volumeToAttach.VolumeName, volumeToAttach.NodeName)
+				continue
+			}
+
+			if rc.isMultiAttachForbidden(volumeToAttach.VolumeSpec) {
+				nodes := rc.actualStateOfWorld.GetNodesForVolume(volumeToAttach.VolumeName)
```

Comment:
If set pv with accessMode 'ReadWriteOnce'，pod with pv running on node1.
when node1 is down，schedule pod to node2，new pod waitting for attach，but old pod is still not detach if node1 is still down 
isMultiAttachForbidden and GetNodeForVolume will produce
`attachdetach-controller  Multi-Attach error for volume "pvc-d0fde86c-8661-11e9-b873-0800271c9f15" Volume is already used by pod `